internal/network/command_network_link_endpoint_create.go

```diff
@@ -0,0 +1,78 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	networkingv1 "github.com/confluentinc/ccloud-sdk-go-v2/networking/v1"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newNetworkLinkEndpointCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <name>",
```

Comment:
`display_name` is optional in the [API spec](https://docs.confluent.io/cloud/current/api.html#tag/Network-Link-Endpoints-(networkingv1)/operation/createNetworkingV1NetworkLinkEndpoint).

I recently made an update to other sub-commands in https://github.com/confluentinc/cli/pull/2451 to make the `name` arg as optional. Please take a look at this.

You would need to update the `network-link service create` command as well. Feel free to do it in another PR.

Reply:
Updated to optional endpoint display name. 

The backend for `network-link service create` actually requires a display name, will update this once I change backend to match API spec.